datapackage.json

```diff
@@ -14,8 +14,8 @@
   ],
   "sources": [
     {
-      "name": "UN Comtrade Reference Tables",
-      "path": "https://unstats.un.org/unsd/tradekb/Knowledgebase/50039/UN-Comtrade-Reference-Tables",
+      "name": "UN Comtrade data extraction API",
+      "path": "https://comtrade.un.org/data/doc/api/",
```

Comment:
Should this link be updated in the main [README.md](https://github.com/datasets/harmonized-system/blob/e739a58b8dad763bc4b58958f89fa90897db0c35/README.md) as well?

Reply:
Yes ir ahould. After this change we can merge it @hlmartin

Reply:
Fixed.